core/core.go

```diff
@@ -269,6 +262,22 @@ func initNetworkID(networkID int, meshDB *meshdb.MeshDB) error {
 }
 
 func (app *App) Start(ctx context.Context) error {
+	// Initialize the p2p node.
+	nodeConfig := p2p.Config{
```

Comment:
Because `p2p.New` requires a context, we needed to move it into `core.App.Start`.